dev/com.ibm.ws.session/src/com/ibm/ws/session/SessionContext.java

```diff
@@ -424,7 +424,7 @@ public int getReaperInterval(long timeout) {
         return rpi;
     }
 
-    protected void setSIPCookieIfApplicable(HttpServletRequest _request, HttpServletResponse response, SessionData sd) {
+    protected void setSIPCookieIfApplicable(HttpServletRequest _request, HttpServletResponse response, AbstractSessionData sd) {
```

Comment:
Can these stay SessionData.  It won't be any other implementation besides that one right?